pagopa/io-dev-api-server
# chore: add ability to activate trials from `config.json`

## Short description
This PR adds ability to activate trials from `config.json` file

## List of changes proposed in this pull request
- Fixed typo in trial system definitions
- Added `trials: t.record(TrialId, SubscriptionState)` to config type
- Added `loadTrials` function to add trials from configuration file at startup

## How to test
Add this configuration to your `config.json` and check that the trails is being added to the trials object
```
{
  ....
  "features": {
    ...
    "trials": {
      "01J2GN4TA8FB6DPTAX3T3YD6M1": "ACTIVE"
    },
  }
}
```
